lib/rubocop/cop/bundler/insecure_protocol_source.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bundler
+      # The source `:gemcutter`, `:rubygems` and `:rubyforge` are deprecated
+      # because HTTP requests are insecure. Please change your source to
+      # 'https://rubygems.org' if possible, or 'http://rubygems.org' if not.
+      #
+      # @example
+      #   # bad
+      #   source :gemcutter
+      #   source :rubygems
+      #   source :rubyforge
+      #
+      #   # good
+      #   source 'https://rubygems.org' # strongly recommended
+      #   source 'http://rubygems.org'
```

Comment:
Seems to me this should be flagged as well. As should any source referred by a symbol or starting with http://.

Reply:
I guess if we do this auto-correction will be limitted, but I think the cop would be more useful it were more generic.

Reply:
I see. I have reasons why I can not make a decision to replace all of `source` with `http://` to `https://`. For example, when specifying an internal gem server using HTTP on the intranet, a use case where HTTPS can not be specified was considered. For this reason, this cop replaces only the special case rubygems.org with `http://` to `https://`.
What do you think?

Reply:
I'm fine with this if the class documentation explains all of this, so people wouldn't wonder why some sources are flagged and others are not.

Reply:
I updated the class documentation with 48676d6.